backtrader/trade.py

```diff
@@ -56,7 +56,7 @@ class TradeHistory(AutoOrderedDict):
     '''
 
     def __init__(self,
-                 status, dt, barlen, size, price, value, pnl, pnlcomm, tz):
+                 status, dt, barlen, size, price, value, pnl, pnlcomm, tz, event=None):
```

Comment:
could you please elaborate on why the `event` parameter is needed and who is responsible to set it.

Reply:
It is needed because unpickle will provide it. During regular backtrader operation it will not be used, that is why I put the default to None. But the __reduce__ method that unpickle will call will provide the event that was once in the object and was pickled.

Reply:
Gr8. Thanks for educating.